service/src/test/java/uk/gov/hmcts/reform/fpl/service/CaseServiceTest.java

```diff
@@ -0,0 +1,51 @@
+package uk.gov.hmcts.reform.fpl.service;
+
+import org.apache.commons.lang3.RandomStringUtils;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import uk.gov.hmcts.reform.authorisation.generators.AuthTokenGenerator;
+import uk.gov.hmcts.reform.ccd.client.CaseUserApi;
+import uk.gov.hmcts.reform.ccd.client.model.CaseUser;
+import uk.gov.hmcts.reform.fpl.enums.CaseRole;
+
+import java.util.Set;
+
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+@ExtendWith(SpringExtension.class)
+public class CaseServiceTest {
+
+    private static final String AUTH_TOKEN = "Bearer user token";
+    private static final String SERVICE_TOKEN = "Bearer service token";
+
+    @Mock
+    private AuthTokenGenerator authTokenGenerator;
+
+    @Mock
+    private CaseUserApi caseUserApi;
+
+    @InjectMocks
+    private CaseService caseService;
+
+    @Test
+    public void shouldAddUserToCase() {
```

Comment:
this can be package-private, same for the class itself

Reply:
done